bundles/org.openhab.binding.radiobrowser/README.md

```diff
@@ -0,0 +1,92 @@
+# Radio Browser Binding
+
+Radio Browser is a community driven database of internet radio and TV stations, that has an open API that is free for all to use and has multiple servers to ensure up time is high.
+With this binding you can use their database of more than 45,000 stations to apply filters and quickly find internet radio streams that interest you.
+
+## Supported Things
+
+- `radio`: Add one of these manually, and it should come online after fetching language and country data to populate the filter channels.
+
+### `radio` Thing Configuration
+
+| Name          | Type    | Description                                                                                 | Default | Required | Advanced |
+|---------------|---------|---------------------------------------------------------------------------------------------|---------|----------|----------|
+| filters       | text    | Allows you to specify your own filters from the `advanced search` of the API.           | `?hidebroken=true&limit=1700&order=clickcount`                                                                          | yes      | no      |
+| clicks        | boolean | Helps to support the server recommend good results.                                         | true    | yes      | yes      |
+| languageCount | integer | If you want less languages to be shown as a filter, you can raise this or create your own.  | 14      | yes      | yes      |
+
+## Filters Configuration
+
+Found by ticking the `show advanced` box, this configuration called `filters` can be used to limit the stations based on codecs, minimum quality, ordering and more.
+The default filter is this and can be change to create better results for your needs.
+
+```
+?hidebroken=true&limit=1700&order=clickcount
```

Comment:
I know it is an advanced channel, but for users it would also be simplify this configuration like `key1=value1,key2=value2` and validate/construct an uri parameter string from that.

Reply:
I agree it would make it slightly more simple so have done it, but do not believe it was worth the work changing it and then redoing documentation that is already written for the small gain.